serverless.yml

```diff
@@ -45,7 +45,7 @@ custom:
     start:
       port: 8008
       inMemory: true
-      migrate: true
+      migrate: false
```

Comment:
don't normally need to alter these

Reply:
Already reverted. Learned a lesson the hard way. This causes build process to fail.

Reply:
Wait, we keep `migrate` as `false` for services. Why does this one need to have it set to `true`?

Reply:
Still not fixed?